tests/Testcontainers.Weaviate.Tests/WeaviateContainerTest.cs

```diff
@@ -0,0 +1,28 @@
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using DotNet.Testcontainers.Commons;
+using Xunit;
```

Comment:
The same applies to tests too. Each module tests has it's own `Usings.cs` file. please see an example [here](https://github.com/testcontainers/testcontainers-dotnet/blob/develop/tests/Testcontainers.PostgreSql.Tests/Usings.cs)